code/integration-tests/local-integration-tests/src/cross_chain_transfer.rs

```diff
@@ -824,11 +825,13 @@ fn trap_assets_larger_than_ed_works() {
 			);
 		// TODO: if we do not top up account initially, than any deposit_creating do not create
 		// anything may be something with zero block or like - fix it better way
+		// -> I don't understand what's the action here
```

Comment:
need to recheck test. 
but idea is that non deposited assets trapped.
but if account of trap non existing for asset - it will not trap (because amount is lower than ED and cannot create account at all). 
better to clean up and split test

Reply:
So it is question if need to init account with some assets on first ever transfer (these are non user account, parachains/relaychain/treasury accounts) or somehow allow well known account to ignore ED requirement.

Reply:
TLRD; if Treasury or other blockchain proxy account does not exist, and we tries to deposit (on transfer) or tip with fee (into treasury) it will fail. so this is bug. at least treasury must be ED 1 for all currencies IMHO. this is kind of bug/issue - can look how Acala/Statemine handle it (but test originally was from Acala)

Reply:
cc @vimukthi-git - not urgent, but may be an issue